examples/uptime.py

```diff
@@ -17,7 +17,7 @@ def get_timeout():
     return ticks_per_second * refresh_interval
 
 def get_msg():
-    val = subprocess.check_output(["uptime", "-p"])
+    val = subprocess.check_output(["uptime", "-p"]).decode("utf-8")
```

Comment:
Yeah, in Python 3 check_output returns a bytes object rather than a string, so the later val.replace calls generate errors because it expects you to provide bytes objects to replace rather than strings.

This just converts it to a string (unicode string in Python 2), so the replace calls work.